docs/tutoriales/tutorial-react.md

```diff
@@ -0,0 +1,237 @@
+---
+id: tutorial-react
+title: React Js
+sidebar_label: React Js
+---
+
+
+React es una librería de JavaScript que servirá para construir interfaces web y aplicaciones móviles con una extensión llamada [React Native](https://reactnative.dev/). React trabaja con unidades de código que se trabajan de forma independiente. React también utiliza una herramienta llamada JSX que permite una interacción muy sencilla con JavaScript, es decir se podrá utilizar con HTML también de forma sencilla.
+>[Más Información](https://es.reactjs.org/)
+
+## Herramientas 
+
+Para React se sugieren las siguientes herramientas:
+
+* Navegador Web (Google Chorme, Moxilla Firefox)
+* [Node Js](https://nodejs.org/es/)
+* Administradores de Paquetes ([Brew](https://brew.sh/index_es), [Chocolatey](https://chocolatey.org/) )
+* Terminal, se puede utilizar con la terminal nativa pero te recomendamos [iTerm2](https://iterm2.com/) , [Hyper](https://hyper.is/) , [cmder](https://cmder.net/)
+* Editor de código, se sugiere que se utilice [Visual Studio Code](https://code.visualstudio.com/)
+
+## Extensiones
+
+Estas extensiones son una recomendación para mejorar tu experiencia en el desarrollo con React.
+
+* Instalar [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=es), esta se encuentra en la chrome web store (Navegador Web)
+* Debugger for Chrome (Visual Studio Code)
+* React Extension Pack (Visual Studio Code)
+* Auto Close Tag (Visual Studio Code)
+* Prettier - Code Formatter (Visual Studio Code)
+
+
+* * *
+## Verificar Instalaciones
+
+Verificar Node js, con estos comandos se puede verificar la versión y además si están instalados.
+
+```shell
+$ node -v
+#v14.15.2
+$ npm -v
+#6.14.9
+```
+
+## Iniciar la primera aplicación con React
+
+Vamos a crear una primera aplicación con React en donde se van a ver reflejados algunas de las funcionalidades de React, en este caso debes cambiar "nombre_de_tu_aplicación" por el nombre que desees, para hacer más sencilla la lectura de ahora en adelante se utilizara my-app.
+
+```shell
+$ npx create-react-app "nombre_de_tu_aplicación"
+```
+Ahora debes entrar a tu directorio de my-app y arrancar tu aplicación.
+
+```shell
+$ cd my-app
+$ yarn start
+```
+Para ver tu código debes utilizar, duplicando tu terminal, lo siguiente: 
+
+```shell
+$ code .
+```
+
+## Primeros Pasos
+
+Una vez abierto el código en tu editor, en este caso recomendablemente Visual Studio Code, puedes navegar hasta encontrar dentro de la carpeta *src* el archivo `App.js` con un contenido similar a este:
+```javascript
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```

Comment:
In this case i dont want to change this code because is a copy-paste default code generated by React JS new proyect, so it can be exactly the same for the tutorial purpose

Reply:
perfect!